blue-note/patterns/media-text.php

```diff
@@ -6,8 +6,8 @@
  * Block Types: core/media-text
  */
 ?>
-<!-- wp:media-text {"align":"wide","mediaId":146,"mediaLink":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/John_Coltrane.png","mediaType":"image","mediaWidth":41,"verticalAlignment":"top","imageFill":true,"style":{"spacing":{"padding":{"right":"0","left":"0"}}}} -->
```

Comment:
This pattern is showing up broken on the site editor, but the code changes make sense to me, there must be something else

Reply:
I tested and the problem is with removing the id of the media-text block. Maybe we should open a Gutenberg issue to see why the block markup fails without them. Let's return them so the pattern goes back to working

Reply:
Yeah I just checked with id  and it works fine.

I will revert this pattern's id and class related changes.

Reply:
I have added Image id & class in this pattern & now its working fine for me.

Reply:
Fantastic, great job!